### ISSUE TITLE
Use of deprecated tap for homebrew in bootstrap.sh

### ISSUE THREAD

**Issue by equal-l2** [redox-os/redox#871]
Hello.

I tried to compile Redox by `bootstrap.sh`, but it forced me to tap `homebrew/versions`.
`homebrew/versions` is now deprecated and should be removed from `bootstrap.sh`.
The problem is that `glendc/gcc_cross_compilers` depends on `gcc49` in `versions`.

Could you give me any idea?

CC:@GlenDC (Owner of `glendc/gcc_cross_compilers`)

**Comment by equal-l2** [comment]
Well... core tap has `gcc@4.9`.
Probably what I should do is to tell GlenDC to use it.

**Comment by GlenDC** [comment]
I merged your change in, you probably still should update the bootstrap.sh script though.

By the way, I also opened a PR with your change in upstream (from which `glendc/homebrew-gcc_cross_compilers` is forked). Once that's merged in there as well, it might be better to reference that repo again, as he seems like a good maintainer. The only reason I was using the fork directly in the bootstrap script is because I didn't know back then how active he/she was, and I didn't want it to block any progress in redox for MacOS users. But he reacted very quickly.

For now it doesn't matter, but at the moment I am not really using that tap for anything myself, so I have little incentive to pro-actively fix these things. Just saying.

Thanks for the effort though! 👍
